pandapipes/io/file_io.py

```diff
@@ -120,10 +121,10 @@ def from_json(filename):
     else:
         with open(filename) as fp:
             json_string = fp.read()
-    return from_json_string(json_string)
+    return from_json_string(json_string, convert=convert)
 
 
-def from_json_string(json_string):
+def from_json_string(json_string, convert=False):
```

Comment:
Why is the default True in from_json and False in from_json_string?

Reply:
This is how pandapower did it. Therefore, I did it the same way.